core/storage/classifier/gae_models.py

```diff
@@ -131,6 +140,7 @@ def create(
             cls, algorithm_id, interaction_id, exp_id, exp_version,
             next_scheduled_check_time, training_data, state_name, status,
             algorithm_version):
+        # type: (Text, Text, Text, int, datetime.datetime, Union[Dict[Text, Any], List[Dict[Text, Any]]], Text, Text, int) -> Text
```

Comment:
Please split this up into multiple lines.

Reply:
Done